spring-boot-project/spring-boot/src/main/java/org/springframework/boot/builder/SpringApplicationBuilder.java

```diff
@@ -97,6 +97,10 @@ public SpringApplicationBuilder(Class<?>... sources) {
 		this.application = createSpringApplication(sources);
 	}
 
+	public SpringApplicationBuilder(ResourceLoader resourceLoader, Class<?>... sources) {
```

Comment:
The other constructor should call this constructor with a `null` `ResourceLoader` (to mirror what  we do in `SpringApplication`.